### PR TITLE
4690 Show UI elements for resource specific metadata editing

### PR DESCRIPTION
**Pull request by pkdash**
**NOTE**: This a fix for a regression bug and needs to go in release 1.57.

### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [x] Peer Code review and approval

### Positive Test Case
1. Create a Web App Connector.
2. Verify that you can see the section "Resource Specific'  in editing mode of the web app connector. Verify that you can edit metadata such as 'Supported Resource Types' or other web app connector specific metadata.

### CODE REVIEW

File: hs_core/templates/pages/genericresource.html [hunk 1]
```diff
@@ -926,25 +926,22 @@ <h4 class="modal-title">Confirm files deletion</h4>
...
-                                {% endif %}
+            {% if cm.resource_type != "CompositeResource" and cm.resource_type != "CollectionResource" %}
+                <div class="col-xs-12 content-block">
+                    <h3>Resource Specific</h3>
+                    <div id="resourceSpecificTab">
+                        <div class="row">
+                            {% if not resource_edit_mode or cm.raccess.published %}
+                                {% block extended_metadata %}{% endblock %}
+                            {% else %}
+                                {% crispy metadata_form  %}
```
Comment (devincowan) [inline hs_core/templates/pages/genericresource.html:937]: In view mode, has the list of supported aggregation types always just been jumbled? It's a bit hard to read, but looks like this wasn't part of this regression...
image

**Review by devincowan — APPROVED** [review]
good catch sir!

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4691](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4691)

File: hs_core/templates/pages/genericresource.html (hunk 1 above)
Comment (devincowan) [reply hs_core/templates/pages/genericresource.html:937]: sounds good https://github.com/hydroshare/hydroshare/issues/4692

**Pull request merged**